core/src/main/java/org/apache/carbondata/core/datastore/page/ColumnPage.java

```diff
@@ -22,22 +22,38 @@
 import java.util.BitSet;
 
 import org.apache.carbondata.core.constants.CarbonCommonConstants;
-import org.apache.carbondata.core.datastore.page.statistics.ColumnPageStatistics;
```

Comment:
Yes, you are right. To make this PR smaller, I suggest to do it in another PR (EncodingStrategy for dimension)